RHEL/7/input/xccdf/system/auditing.xml

```diff
@@ -770,7 +775,10 @@ will alert the system administrator(s) to any modifications. Any unexpected
 users, groups, or modifications should be investigated for legitimacy.</rationale>
 <ident cce="27192-4" />
 <oval id="audit_rules_usergroup_modification" />
-<ref nist="AC-2(4),AC-17(7),AU-1(b),AU-2(a),AU-2(c),AU-2(d),AU-12(a),AU-12(c),IR-5" disa="18,172,1403,1404,1405,1684,1683,1685,1686" ossrg="476,239" pcidss="Req-10.2.5" cis="5.2.5" />
+<ref nist="AC-2(4),AC-17(7),AU-1(b),AU-2(a),AU-2(c),AU-2(d),AU-12(a),AU-12(c),IR-5" 
+disa="18,172,1403,2130" 
+stigid="030730" pcidss="Req-10.2.5" cis="5.2.5"
```

Comment:
On 4/15/16 5:08 PM, redhatrises wrote:
>
> In RHEL/7/input/xccdf/system/auditing.xml 
> <https://github.com/OpenSCAP/scap-security-guide/pull/1211#discussion_r59941157>:
>
> > @@ -770,7 +775,10 @@ will alert the system administrator(s) to any modifications. Any unexpected
> >  users, groups, or modifications should be investigated for legitimacy.</rationale>
> >  <ident cce="27192-4" />
> >  <oval id="audit_rules_usergroup_modification" />
> > -<ref nist="AC-2(4),AC-17(7),AU-1(b),AU-2(a),AU-2(c),AU-2(d),AU-12(a),AU-12(c),IR-5" disa="18,172,1403,1404,1405,1684,1683,1685,1686" ossrg="476,239" pcidss="Req-10.2.5" cis="5.2.5" />
> > +<ref nist="AC-2(4),AC-17(7),AU-1(b),AU-2(a),AU-2(c),AU-2(d),AU-12(a),AU-12(c),IR-5"
> > +disa="18,172,1403,2130"
> > +stigid="030730" pcidss="Req-10.2.5" cis="5.2.5"
>
> Is this supposed to be |030710|?
>
Yes - good catch! Updated patch in flight.

-- 
Shawn Wells
Chief Security Strategist
U.S. Public Sector
shawn@redhat.com | 443.534.0130